contain-intrinsic-size.bs

```diff
@@ -0,0 +1,125 @@
+<pre class='metadata'>
+Title: CSS Contain Intrinsic Size
+Shortname: css-contain-intrinsic-size
+Level: 1
+Status: UD
+Group: csswg
+Work Status: exploring
+URL: https://wicg.github.io/display-locking/contain-intrinsic-size.html
+Editor: Tab Atkins-Bittner, Google
+Editor: Vladimir Levin, Google
+Abstract: This spec introduces a contain-intrinsic-size property, which allows
+          authors to specify element's intrinsic size when [=size containment=]
+          applies.  In effect, this feature extends [=size containment=] by 
+          letting the developer specify the placeholder size to apply when the
+          element's children are not considered.
+</pre>
+
+Introduction {#intro}
+============
+
+When [=size containment=] is specified, the element does not consider its
+children when determining its intrinsic size, which in turn can affect the
+element's [=principal box=] size. This is benefitial for performance, as it can
+allow the user-agent to skip processing layout of elements in the size
+contained subtrees. However, it also makes it difficult to create responsive
+design content, since the element needs to be sized for it to take up layout
+space.
+
+The proposed property, 'contain-intrinsic-size', allows the developer to
+specify a size that replaces the intrinsic size when [=size containment=] is
+specified. This means that [=size containment=] can be toggled on and off, with
+the element either considering the specified value or its children for
+intrinsic sizing. This is particularly helpful for features such as <a
+href="https://wicg.github.io/display-locking/#subtree-visibility-auto">
+'subtree-visibility: auto'</a>, which applies [=size containment=]
```

Comment:
It also helps with other values of subtree-visibility.

Reply:
Makes sense, I'll update the wording here.